install/README.md

```diff
@@ -162,22 +162,30 @@ types include Slack, Pager Duty, email, or a custom integration via a generic we
 See [Configure Prometheus Email Alert Notifications](../operations/system_management_health/Configure_Prometheus_Email_Alert_Notifications.md) for an example
 configuration of an email alert notification for the Postgres replication alerts that are defined on the system.
 
-### 8. Update firmware with FAS
+### 8. Upload Olympus BMC recovery firmware into TFTP server
```

Comment:
You need to update the table of contents to include this new section, and update the links to point to the new header names: https://github.com/Cray-HPE/docs-csm/blob/d0a8596ac958e3482c2fc8f114c072b312a59ad3/install/README.md?plain=1#L42